pkg/resources/management.cattle.io/v3/cluster/Cluster.md

```diff
@@ -0,0 +1,5 @@
+## Cluster
```

Comment:
We don't need to specify cluster here because that gets generated in docs.md
```suggestion
## Validation Checks
```

Reply:
Agreed.